orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -381,6 +381,22 @@ OrtValue CastToType(onnxruntime::ORTInvoker& invoker, const OrtValue& input, at:
   return output[0];
 }
 
+void CastToTypeInPlace(onnxruntime::ORTInvoker& invoker, const OrtValue& input, OrtValue& output, at::ScalarType type) {
```

Comment:
I'm confused by the name `InPlace`.  It takes a `const` input as well as an output.  The Invoke call is taking an std::move of the input, so it seems like it's not happening in place. What am I missing?

Reply:
Just looked at the header and saw the existing Cast function. Maybe following the aten convention this should be `Cast_Out` or similar?